### ISSUE TITLE
Two resting state scans: before and after TMS 

### ISSUE THREAD

**Issue by dkp** [MICA-MNI/micapipe#48]
We have two runs of rsfmri in a single session:

```
tree inputs/sub-219/ses-itbs/func
inputs/sub-219/ses-itbs/func
├── sub-219_ses-itbs_task-rest_run-01_bold.json
├── sub-219_ses-itbs_task-rest_run-01_bold.nii.gz
├── sub-219_ses-itbs_task-rest_run-01_events.tsv
├── sub-219_ses-itbs_task-rest_run-02_bold.json
├── sub-219_ses-itbs_task-rest_run-02_bold.nii.gz
└── sub-219_ses-itbs_task-rest_run-02_events.tsv
```

I'd like to process them separately, so I specified and ran two separate docker commands:

```
docker run -ti --rm \
    -v ${PWD}/inputs:/bids_dataset:ro \
    -v ${PWD}/outputs:/output_directory \
    -v ${PWD}/working:/tmp \
    -v ${PWD}/inputs/code/freesurfer_license_file.txt:/opt/freesurfer-6.0.0/license.txt \
    micalab/micapipe:latest \
    -bids /bids_dataset \
    -out /output_directory \
    -sub 219 -ses itbs \
    -proc_rsfmri \
    -mainScanStr task-rest_run-01_bold -mainScanRun 1
```

and 

```
docker run -ti --rm \
    -v ${PWD}/inputs:/bids_dataset:ro \
    -v ${PWD}/outputs:/output_directory \
    -v ${PWD}/working:/tmp \
    -v ${PWD}/inputs/code/freesurfer_license_file.txt:/opt/freesurfer-6.0.0/license.txt \
    micalab/micapipe:latest \
    -bids /bids_dataset \
    -out /output_directory \
    -sub 219 -ses itbs \
    -proc_rsfmri \
    -mainScanStr task-rest_run-02_bold -mainScanRun 2
```
Only the first run claims to have completed.  How can I get separate output for each run?

**Comment by dkp** [comment]
Follow up: I ran run-01 first and then renamed the func directory, then I ran run2.
I guess this works.

**Comment by rcruces** [comment]
Hi, you found a great solution to this. As we also encountered a similar issue we implemented a different solution to process multiple functional acquisitions in the same derivatives output directory. The code is part of the `0.1.3` release, we are currently building the container and the readTheDocs will be updated with this new functionality. The new modality is called `-proc_func` and it allows to process multiple functional acquisitions (not at the same time) as well al multi echo acquisitions. I'll tell you when the container is ready
